sympy/sets/tests/test_sets.py

```diff
@@ -1487,3 +1487,20 @@ def test_issue_16878b():
     # that handles the base_set of S.Reals like there is
     # for Integers
     assert imageset(x, (x, x), S.Reals).is_subset(S.Reals**2) is True
+
+def test_external_disjoint_union():
+    assert external_disjoint_union(FiniteSet(1,2,3), FiniteSet(1,2,3), FiniteSet(1,2,3)) == (FiniteSet(1,2,3)*FiniteSet(0, 1,2))
```

Comment:
Please make one spacing between the commas for PEP8.

Reply:
I'm on it. While computing the disjoint union, if there are some empty sets in the arguments, should I ignore them (unless all arguments are empty sets), because that would just use up an index. For eg:

>>> from sympy import *
>>> external_disjoint_union(FiniteSet(1,2), EmptySet, FiniteSet(3,4))
Union(ProductSet(FiniteSet(1, 2), FiniteSet(0)), ProductSet(FiniteSet(3, 4), FiniteSet(2)) 

So here, in the final result, no element has index = 1. Should I leave this as a feature, i.e., the user can get the index they want by supplying the appropriate number of empty sets in between the non empty arguments?

Reply:
This can be an open question. 
But I'd keep the behavior for the empty set such that users can shift the index by giving the empty set.